### ISSUE TITLE
Flaky TableReaderTailRecordCursorTest.testByYear()

### ISSUE THREAD

**Issue by bluestreak01** [questdb/questdb#1515]
### Describe the bug

Failed once on JDK8 Linux CI:

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f121dbf81f4, pid=2700, tid=0x00007f11ee6ec700
#
# JRE version: OpenJDK Runtime Environment (8.0_292-b10) (build 1.8.0_292-b10)
# Java VM: OpenJDK 64-Bit Server VM (25.292-b10 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# J 4283 C2 io.questdb.cairo.TableReaderTailRecordCursorTest.lambda$null$1(Ljava/util/concurrent/CyclicBarrier;ILjava/util/concurrent/atomic/AtomicInteger;Ljava/util/concurrent/CountDownLatch;)V (289 bytes) @ 0x00007f121dbf81f4 [0x00007f121dbf77a0+0xa54]
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# An error report file with more information is saved as:
# /home/vsts/work/1/s/core/hs_err_pid2700.log

```

### Environment

```markdown
- **QuestDB version**:
- **OS**:
- **Browser**:
```

**Comment by ideoma** [comment]
duplicate of #1556, likely to be fixed in #1557
